src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -71,6 +74,14 @@
 	private MappingSet[] srcMappingCache = new MappingSet[2];
 	private Mercury[] srcMercuryCache = new Mercury[2];
 
+	/**
+	 * Loom will generate a new genSources task (with a new name, based off of {@link LoomDecompiler#name()})
+	 * that uses the specified decompiler instead.
+	 */
+	public void addDecompiler(LoomDecompiler decompiler) {
```

Comment:
I think at somepoint we can come up with a better api for this, but for now this will do fine.